plugin.php

```diff
@@ -1,19 +1,25 @@
 <?php
 
 /**
- * Plugin Name: Delegated OAuth 2 Authentication
- * Description: Delegate authentication to another OAuth 2 WordPres site.
+ * Plugin Name: Delegated Authentication
+ * Description: Delegate authentication to another OAuth 2 WordPress site.
  * Author: Joe Hoyle
  */
-
-namespace HM\Delegated_OAuth2;
+namespace HM\Delegated_Auth;
 
 require_once __DIR__ . '/inc/namespace.php';
+require_once __DIR__ . '/inc/oauth2/namespace.php';
+require_once __DIR__ . '/inc/cookie/namespace.php';
 
-if ( ! defined( 'HM_DELEGATED_OAUTH2_REST_BASE' ) ) {
+if ( ! defined( 'HM_DELEGATED_AUTH_REST_BASE' ) ) {
```

Comment:
Could we have a default for this instead?

Reply:
Hmm I don't think so. This plugin shouldn't be tied to any upstream oauth2 WordPress instance.